File: Python/Template.py
from typing import List

class Solution:
    def method(self, x) -> None:
        return


if __name__ == "__main__":
    sol = Solution()
    tests = [
        
    ]
    
    passed_all = True
    test_only = 0
    for i, test in enumerate(tests, 1):
        if test_only and test_only != i:
            continue
        x, expected = test
        actual = sol.method(x)
        if actual != expected:
            print(f"Test {i} Failed")
            print(f"\tActual  : {actual}")
            print(f"\tExpected: {expected}")
            passed_all = False
    
    if passed_all:
        print("All Tests Passed")

# ---------------------------- Other Solutions --------------------------------

File: Go/Template.go
package main

import "fmt"

func function(nums []int) []int {
	return nums
}

func print_failed_test(i int, res []int, expected []int) {
	fmt.Printf("Test %d failed:\n", i+1)
	fmt.Printf("    \tActual  : %v\n", res)
	fmt.Printf("    \tExpected: %v\n", expected)
}

func main() {
	tests := []struct {
		nums     []int
		expected []int
	}{
		{[]int{1, 2, 3, 1}, []int{1, 2, 3, 1}},
		{[]int{1, 2, 3, 4}, []int{1, 2, 3, 4}},
		{[]int{1, 1, 1, 3, 3, 4, 3, 2, 4, 2}, []int{1, 1, 1, 3, 3, 4, 3, 2, 4, 2}},
	}

	passed_all := true
	for i, test := range tests {
		nums, expected := test.nums, test.expected
		res := function(nums)
		if len(res) != len(expected) {
			passed_all = false
			print_failed_test(i, res, expected)
			continue
		}
		for j, num := range res {
			if num != expected[j] {
				passed_all = false
				print_failed_test(i, res, expected)
				break
			}
		}
	}
	if passed_all {
		fmt.Println("Passed all tests.")
	}
}
